cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -885,6 +905,12 @@ mod test {
         s[..off].lines().count()
     }
 
+    fn check_errors(errs: &[YaccGrammarError], errs_expect: &[YaccGrammarError]) -> bool {
```

Comment:
Function needs a doc string.

Reply:
I went ahead and pushed a fix for this, but it will likely be impacted by the field removal of `duplicate_*` discussion,
since this is just avoiding ordering instability.  If we switch to an `errors` field, that will likely go away.